source/daq/triggered_writer.cc

```diff
@@ -42,16 +42,18 @@ namespace psyllid
             f_center_freq( 50.e6 ),
             f_freq_range( 100.e6 ),
             f_monarch_ptr(),
-            f_stream_no( 0 )
+            f_stream_no ( 0 )
```

Comment:
I had to add f_stream_no again. It was still used by the triggered_writer. Do we want to remove/replace it completely?